app/src/util/tests/renderWithProviders.tsx

```diff
@@ -16,7 +16,9 @@ const renderWithProviders = (component: React.ReactElement, withStore?: Store) =
   const result = render(
     <StoreProvider store={store}>
       <ThemeProvider>
-        <Router history={store.router.history}>{component}</Router>
+        <Suspense fallback={<></>}>
```

Comment:
With the refactor of `Routes` (renamed `AppRoutes`) `Suspense` and `Router` are now in the `App` component. In the absence of `Suspense` and `Router`, `AppRoutes.spec.tsx` throws `A React component suspended while rendering, but no fallback UI was specified.` error. Adding `Suspense` here to address that issue and shouldn't affect other components.